manifests/integreatly-3scale/3scale-0.4.0/3scale-operator.v0.4.0.clusterserviceversion.yaml

```diff
@@ -1,95 +1,95 @@
 apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
-  name: 3scale-operator.v1.9-7
+  name: 3scale-operator.v0.4.0
```

Comment:
Why is the version here (and in the filenames) v0.4.0, when the actual operator image has a tag of 1.10-4. Should this say v1.10.0 or v1.10.4 or something?

Reply:
`0.4.0` is the operator version as specified in `version.go` but the tagged release is `1.10-4`. Not too sure what the convention for these versions are since this is mostly a straight copy from the 3scale repo. I had asked @matskiv about it and he said can just leave it as is :man_shrugging: 
Unsure how this would fit it with DeLorean eventually also if we'd always need to change the versions / filenames to match the release tag version instead :confused:

Reply:
I've just created this issue to see if we can get clarification on that: https://github.com/3scale/3scale-operator/issues/297

Of course, we don't need to wait for a reply there to continue here. :slightly_smiling_face:

Reply:
I've just created this issue to see if we can get clarification on that: https://github.com/3scale/3scale-operator/issues/297

Of course, we don't need to wait for a reply there to continue here. :slightly_smiling_face:

Reply:
This can be helpful
https://github.com/3scale/3scale-operator/blob/2.7-stable-prod/deploy/olm-catalog/3scale-operator/3scale-operator.v0.4.0.clusterserviceversion.yaml